src/dstack/_internal/cli/commands/pool.py

```diff
@@ -298,10 +298,14 @@ def _add_ssh(self, args: argparse.Namespace) -> None:
             try:
                 network = ipaddress.ip_network(args.network)
```

Comment:
The nested try-except-try statements look redundant. Consider removing this outer try.
```python
try:
    network = ipaddress.IPv4Interface(args.network).network
except ValueError as e:
    console.print(...)
    return
```

Reply:
Many thanks! That made code better!